src/tools/kata-ctl/README.md

```diff
@@ -0,0 +1,13 @@
+# Kata Control tool
+
+## Overview
+
+## Audience and environment
+
+## Full details
```

Comment:
Might be worth adding a "History" sub section so that we can explain that this is the Kata 3.x evolution of the `kata-runtime` utility program.